### PR TITLE
Dataproc removed ANACONDA as an optional component in preview2

### PR DESCRIPTION
**Pull request by sameerz**
Dataproc removed ANACONDA, trying to install with ANACONDA fails.  Removing from the documentation.  

Ref: https://cloud.google.com/dataproc/docs/release-notes#September_30_2020

I will follow up with a PR to branch-0.3 with this and other doc fixes.  

This was already updated in the instructions in the Dataproc repo https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/rapids/README.md#step-1-create-dataproc-cluster-with-spark-rapids-accelerator

### CODE REVIEW

**Comment by sameerz** [comment]
Similar to PR #903 but for the gh-pages branch.

**Comment by jlowe** [comment]
In the future, we should just wait until this is merged to the dev branch and cherry-pick the commit, preserving the author, etc., to the gh-pages branch.

**Review by jlowe — APPROVED** [review]

**Pull request merged**
